### PR TITLE
Hangman quote embeds

### PR DESCRIPTION
**Pull request by ArPiiX**

### CODE REVIEW

File: src/app/commands/hangman.py [hunk 1]
```diff
@@ -20,29 +20,35 @@ def main(system: SystemMessenger, author: User, cmd: Command, params: list):
     # game command should be                            /hangman start *word*
     # then it should be                                 /hangman guess *char*
     # to check game state when connection lost try     /hangman state
+    embed = Embed(title="Hangman", thumbnail="http://simpleicon.com/wp-content/uploads/users.png", color="#00ff00")
```
Comment (zaanposni) [inline src/app/commands/hangman.py]: use a appropriate thumbnail

File: src/app/commands/hangman.py [hunk 2]
```diff
@@ -56,11 +62,14 @@ def main(system: SystemMessenger, author: User, cmd: Command, params: list):
...
+                    embed.set_text(f"{author.display_name} has tried {params[1]}")
+                    system.broadcast(embed)
                     SHL.output(f"{author} hast tried to guess {params[1]} as a char", "HangmanGame")  # log
-                    system.broadcast(hangman_game.check_char(params[1], author))
+                    embed.set_text(hangman_game.check_char(params[1], author))
+                    system.broadcast(embed)
                     return
-                system.broadcast(f"{author.display_name} the filthy cheater tried to guess their own word!")
+                embed.set_text(f"{author.display_name} the filthy cheater tried to guess their own word!")
+                system.broadcast(embed)
```
Comment (zaanposni) [inline src/app/commands/hangman.py]: on none-success events do not use `00ff00` as color. -> use red or orange.

File: src/app/commands/hangman.py [hunk 3]
```diff
@@ -74,9 +83,11 @@ def main(system: SystemMessenger, author: User, cmd: Command, params: list):
                 if hangman_game.initiator != author:
                     SHL.output(f"{author} tried to solve {hangman_game.word_clear} with {params[1]}",
                                "HangmanGame")  # log
-                    system.broadcast(hangman_game.check_word(params[1], author))
+                    embed.set_text(hangman_game.check_word(params[1], author))
+                    system.broadcast(embed)
                     return
-                system.broadcast(f"{author.display_name} the filthy cheater tried to solve their own word!")
+                embed.set_text(f"{author.display_name} the filthy cheater tried to solve their own word!")
+                system.broadcast(embed)
```
Comment (zaanposni) [inline src/app/commands/hangman.py]: same here

File: src/app/commands/hangman.py (hunk 1 above)
Comment (ArPiiX) [reply src/app/commands/hangman.py]: the thumbnail in hangman was intentional, as the game is for everyone and the users thumbnail looks good to me in that case

**Pull request merged**